src/runtime-rs/crates/hypervisor/src/dragonball/inner.rs

```diff
@@ -338,13 +348,115 @@ impl DragonballInner {
         Ok((old_vcpus, new_vcpus))
     }
 
+    pub(crate) fn resize_memory(
+        &mut self,
+        old_mem_mb: u32,
```

Comment:
we cloud simply save a balloon size in self, or just force to clear balloon size. It seems that resize_memory() only need a target memory size is enough

Reply:
fixed